account_bank_statement_import_transfer_move/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+ This module allows you to add generation of additional line in bank statement.
```

Comment:
This sentence doesn't say what is the purpose of this additional line in the bank statement?